src/bp-activity/bp-activity-filters.php

```diff
@@ -1737,6 +1737,17 @@ function bp_activity_create_parent_media_activity( $media_ids ) {
 			$activity_id = bp_activity_post_update( array( 'content' => $content ) );
 		}
 
+		// Add in description in attachment when only one document uploaded.
+		if ( ! empty( $activity_id ) && is_array( $media_ids ) && 1 === count( $media_ids ) && $content ) {
```

Comment:
@chetansatasiya can we check if `$content` is empty or not? right now it is only checking if `$content` variable is set.

Reply:
@mehulkaklotar DONE 👍